pkg/master/master.go

```diff
@@ -421,6 +420,25 @@ func (m *Master) init(c *Config) {
 	m.masterServices.Start()
 }
 
+// Adds a service to return the supported api versions.
+func addApiWebService(container *restful.Container, apiPrefix string, enableV1Beta3 bool) {
+	// TODO: InstallREST should register each version automatically
+	versionHandler := apiserver.APIVersionHandler("v1beta1", "v1beta2")
```

Comment:
I can do that. But I am not sure that it is better.
Do you think that passing the list is better?